packages/backend/src/peripherals/arbitrum/multicall/ArbitrumMulticall.ts

```diff
@@ -0,0 +1,100 @@
+import { ChainId, EthereumAddress } from '@l2beat/shared-pure'
+
+import { EthereumClient } from '../../ethereum/EthereumClient'
+import {
+  MulticallEncoder,
+  MulticallRequest,
+  MulticallResponse,
+} from './interfaces'
+import { arbitrumMulticallEncoder } from './MulticallEncoder'
+
+/**
+ * @see https://arbiscan.io/address/0x842eC2c7D803033Edf55E478F461FC547Bc54EB2
+ */
+export const ARBITRUM_MULTICALL_BATCH_SIZE = 150
```

Comment:
can you explain why there is number 150?

Reply:
leave a TODO comment and add task in Linear project "L2 assets full" to investigate this value

Reply:
https://linear.app/l2beat/issue/L2B-1822/investigate-max-batch-size-for-multicall
https://github.com/l2beat/l2beat/pull/1645/commits/769a991f159093ad2c92fdde2f5d539ec3af73b0